gematria/datasets/bhive_importer_test.cc

```diff
@@ -182,27 +182,37 @@ TEST_F(BHiveImporterTest, MultipleInstructions) {
              source: "bhive: skl"
              inverse_throughput_cycles: 2.07
            })pb";
-  EXPECT_THAT(
-      x86_bhive_importer_->ParseBHiveCsvLine(
-          kSourceName, "4829d38b44246c8b54246848c1fb034829d04839c3,207.000000",
-          /*throughput_scaling=*/kScaling, /*base_address=*/100),
-      IsOkAndHolds(EqualsProto(kExpectedBasicBlockProto)));
+  EXPECT_THAT(x86_bhive_importer_->ParseBHiveCsvLine(
+                  kSourceName,
+                  "4829d38b44246c8b54246848c1fb034829d04839c3,207.000000", 0, 1,
+                  /*throughput_scaling=*/kScaling, /*base_address=*/100),
+              IsOkAndHolds(EqualsProto(kExpectedBasicBlockProto)));
 }
 
 TEST_F(BHiveImporterTest, NoCommaOnLine) {
```

Comment:
Let's rename this one to InvalidColumnIndices (and add another failing case with more columns).